src/monitor/Monitor.py

```diff
@@ -0,0 +1,225 @@
+import sys
+import time
+import os
+import pandas as pd
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, NamedTuple, Tuple
+
+if __name__ == "__main__":
+    sys.path.append(str(Path(__file__).resolve().parents[2]))
+    from run_container import run_container
+    from lib.WordOperator import str_format
+    from src.HostInfo import BasicCapability, BookingTime, HostInfo, ScheduleColumnNames, ScheduleDF, UserConfig
```

Comment:
if Monitor.py is not executing as "__main__" it can not import those properties, so please change it as:
```suggestion

from run_container import run_container
from lib.WordOperator import str_format
from src.HostInfo import BasicCapability, BookingTime, HostInfo, ScheduleColumnNames, ScheduleDF, UserConfig
```